src/OpenTelemetry.Contrib.Instrumentation.AWS/README.md

```diff
@@ -17,7 +17,7 @@ using OpenTelemetry.Trace;
 public void ConfigureServices(IServiceCollection services)
 {
     services.AddControllers();
-    services.AddOpenTelemetryTracing((builder) => builder
+    services.AddOpenTelemetry().WithTracing(builder => builder
```

Comment:
I'm not sure if we should simply update the README with the new methods.

It could be the case that a lot of the users of AWS instrumentation still use older version of OTel and the extensions hosting package (consider this project still references OTel SDK 1.0.1)

Reply:
I will revert this file. Other redmes looks good

Reply:
834f46717bd61194b29f14cca4ae497c2672d18c